app/src/main/kotlin/com/jakewharton/sa4p/ui/theme.kt

```diff
@@ -137,6 +138,7 @@ private val DarkColors = darkColorScheme(
 
 @Composable
 fun Theme(
+	modifier: Modifier = Modifier,
```

Comment:
Not sure this needs a modifier. Does `MaterialTheme` take one?

Reply:
MaterialTheme doesn't have a modifier.